unime/src/i18n/en/index.ts

```diff
@@ -243,6 +280,8 @@ const en = {
   CLOSE: 'Close',
   CONTINUE: 'Continue',
   SKIP: 'Skip',
+  REJECT: 'Reject',
+  ACCEPT: 'Accept',
```

Comment:
We're removing a little bit of information by unifying the "Accept connection" and "Accept credentials" on the respective buttons into a more generic "Accept". Is this deliberate?

Reply:
I just noticed that the `$LL.ACCEPT()` is never used anywhere since we have the specific "Accept credentials" and "Accept connection". So this key can be removed as well. 👍